dev-db/etcd/files/etcd2.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=etcd
+
+[Service]
+User=etcd
+PermissionsStartOnly=true
+Environment=ETCD_DATA_DIR=/var/lib/etcd
+ExecStart=/usr/bin/etcd
```

Comment:
That's a good question. @philips, what is the recommended strategy here? Should we use a separate data-dir for etcd2 and ask the user to move their data (or update the service file) if they want to migrate to etcd2? Or just use the same directory?